layouts/index.html

```diff
@@ -76,4 +80,8 @@
   {{ partial "js.html" . }}
 </body>
 
-</html>
\ No newline at end of file
+<<<<<<< HEAD
+</html>
+=======
+</html>
+>>>>>>> 8ae97e6... include hackathon section
```

Comment:
Something might have gone a little awry here with a git conflict. Effectively this change just isn't a change :)